src/Common/geo/WebIpToLocationResolver.cpp

```diff
@@ -133,8 +138,23 @@ void WebIpToLocationResolver::replyFinished(QNetworkReply *reply)
 
         QJsonObject root = doc.object();
         if (!root.contains("country_name")) {
-            qCritical() << "The root json object not contains 'country_name'";
-            return;
+            if (root.contains("error")) {
+                auto error= root["error"].toObject();
+                auto code = error.contains("code") ? error["code"].toInt() : -1;
+                if (retryCount<MaxServersAlternatives)
+                {
+                    qDebug() << "Error " << code << " received, trying alternative server ...";
+                    requestDataFromWebService(ip, retryCount+1);
```

Comment:
I'm not 100% shure, but I think in Qt code style **retryCount + 1** will be **retryCount + 1**, the plus operator is surrounded with spaces.

Reply:
agreed for this one, do you use a code style tool?

Reply:
Agreed